.github/workflows/cpp.yml

```diff
@@ -70,10 +70,7 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - name: Fetch Submodules and Tags
-        run: ci/scripts/util_checkout.sh
-      - name: Free Up Disk Space
-        run: ci/scripts/util_cleanup.sh
+          submodules: recursive
```

Comment:
I noticed that some checkouts have `submodules: recursive` and others don't — is there a pattern here that I'm missing?

Reply:
Correct, we don't run flight/parquet/integration tests everywhere to can spare some additional time by omitting the submodule checkouts.